01/nonickid/wordvalue.py

```diff
@@ -0,0 +1,22 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    with open('../' + DICTIONARY) as fn:
+        return [word.strip() for word in fn.readlines()]
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum([ LETTER_SCORES.get(key.upper(), 0) for key in word])
```

Comment:
you can omit the list comprehension [] in `sum`

Reply:
love `get`

Reply:
Thank you for valuable comment